mathjax3-ts/input/tex/ams_cd/AmsCdMappings.ts

```diff
@@ -0,0 +1,38 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Symbol mappings for the AMScd package.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import * as sm from '../SymbolMap.js';
+import ParseMethods from '../ParseMethods.js';
+import AmsCdMethods from './AmsCdMethods.js';
+
+
+new sm.EnvironmentMap('amsCd_environment', ParseMethods.environment,
+                      {CD: 'CD'}, AmsCdMethods);
+
+new sm.CommandMap('amsCd_macros', {
+  minwidth: 'minwidth',
+  minheight: 'minheight',
+}, AmsCdMethods);
```

Comment:
These are non-standard commands, right?  I checked the documentation, and it seems there is a standard one for the with: `\minCDarrowwidth`, but not one for height.  Since these commands are global, it might be best to have them CD-specific like this, rather than too generic.

Reply:
I've renamed the commands. PTAL.